client/layout/masterbar/item.tsx

```diff
@@ -60,8 +60,7 @@ class MasterbarItem extends Component< MasterbarItemProps > {
 		isOpenForNonMouseFlow: false,
 	};
 
-	componentButtonRef = React.createRef< HTMLButtonElement >();
-	componentDivRef = React.createRef< HTMLDivElement >();
```

Comment:
By always rendering the wrapper, and either and `a` or a `button`, `componentButtonRef`  is not necessary anymore, and `componentDivRef` was renamed to `wapperRef` for better clarity